From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 13 Jul 2021 07:30:20 +0200
Subject: [PATCH] don't build tls tests

They fail to compile because gnutls is built without pkcs11 support in
ptxdist:

.../tls/tests/certificate.c:597: undefined reference to `gnutls_pkcs11_init'
.../tls/tests/certificate.c:598: undefined reference to `gnutls_pkcs11_add_provider'

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 81c2a8b15ca0..1a2c447b21e8 100644
--- a/meson.build
+++ b/meson.build
@@ -189,7 +189,6 @@ if backends.contains('openssl')
   subdir('tls/openssl')
 endif
 
-subdir('tls/tests')
 
 # Will automatically pick it up from the cross file if defined
 gio_querymodules = find_program('gio-querymodules', required : false)
